core/terraform/aws/workshop-core/variables.tf

```diff
@@ -52,4 +52,20 @@ variable "onprem_topics" {
 
 variable "feedback_form_url" {
   description = "Feedback Form Url"
-}
\ No newline at end of file
+}
+
+variable "vpc_cidr_block" {
+  default     = "10.0.0.0/16"
+  description = "VPC CIDR block"
+}
+
+variable "public_subnet_cidr_blocks" {
+  default     = ["10.0.101.0/24", "10.0.102.0/24"]
+  description = "List of public subnet CIDR blocks"
+}
+
+variable "availability_zones" {
+  default     = ["eu-west-2a", "eu-west-2b"]
```

Comment:
mmm... are these configurable? Can we use the region in the properties somehow? @moliele

Reply:
I think we should add these as variables to the workshop-example.yamls. I had a problem with overlapping with an existing subnet.